packages/loader/lib/main.js

```diff
@@ -26,12 +25,13 @@ var mainContext = this;
     };
 
     requirejs = require = requireModule = function(name) {
-      var s = seen[name];
+      var exports = seen[name];
 
-      if (s !== undefined) { return seen[name]; }
-      if (s === UNDEFINED) { return undefined;  }
+      if (exports !== undefined) {
```

Comment:
it will just show up as `{}` in the `seen` map (should call that map exports I think)